templates/ExpenseTrackerContract/.template.config/template.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json.schemastore.org/template",
```

Comment:
All these should be added into aelf-samples repo to maintain consistency

Reply:
Added to aelf-samples